views/partials/_whats-new.njk

```diff
@@ -6,8 +6,8 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-        <p class="govuk-body">16 December 2021: We’ve released GOV.UK Frontend v4.0.0. This ‘breaking release’ contains an update to the accordion and fixes to the footer.</p>
-        <p class="govuk-body"><a href="https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0" class="govuk-link">Read the release notes to find out what changes you need to make to update</a>.</p>
+        <p class="govuk-body">3 February 2022: We’ve released GOV.UK Frontend v4.0.1. This ‘fix release’ contains improvements to the error summary, input and accordion components.</p>
```

Comment:
Should it be 'text input' instead of 'input'?

Reply:
@EoinShaughnessy Yep, we call it a "text input" in the Design System, so best to be consistent.

We've also just merged a fix for the select component - might be worth adding that too. https://github.com/alphagov/govuk-frontend/pull/2524

Reply:
Will do, thanks!

```suggestion
        <p class="govuk-body">3 February 2022: We’ve released GOV.UK Frontend v4.0.1. This ‘fix release’ contains improvements to the error summary, text input, select and accordion components.</p>
```